pkg/server/server.go

```diff
@@ -652,6 +664,9 @@ func (s *ProxyServer) Connect(stream agent.AgentService_ConnectServer) error {
 				return
 			}
 
+			if len(recvCh) > 9 {
+				klog.V(2).InfoS("Receive channel on Connect is full", "agentID", agentID, "serverID", s.serverID)
```

Comment:
nit: maybe warning level?

Reply:
Not if we want to stay with structured logging.
https://github.com/kubernetes/klog/pull/184#issuecomment-717405511